src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IObjectCreationExpression.cs

```diff
@@ -1409,7 +1409,7 @@ void M()
             var expectedDiagnostics = new DiagnosticDescription[] {
                 // file.cs(21,33): error CS9366: The type 'IInterface' may not be used as the target-type of 'new'.
                 //         IInterface iinterface = new() /*<bind>*/{ 1, 2, 3 }/*</bind>*/;
-                Diagnostic(ErrorCode.ERR_BadTargetTypeForNew, "new() /*<bind>*/{ 1, 2, 3 }").WithArguments("IInterface").WithLocation(21, 33)
+                Diagnostic(ErrorCode.ERR_IllegalTargetType, "new() /*<bind>*/{ 1, 2, 3 }").WithArguments("IInterface").WithLocation(21, 33)
```

Comment:
>ERR_IllegalTargetType [](start = 37, length = 21)

This name for the error code is worse than the old name, as the old name gave a hint that it was specific to the `new` construct.